apps/camera/js/views/preview-gallery.js

```diff
@@ -125,6 +130,17 @@ return View.extend({
     this.els.previewMenu.classList.remove('visible');
   },
 
+  onFrameWheel: function(event) {
+    if (event.deltaMode !== event.DOM_DELTA_PAGE || event.deltaY) {
+      return;
+    }
+    if (event.deltaX > 0) {
+      this.fire('swipe', 'left');
```

Comment:
nit: I think we use `emit()` everywhere else, but `fire()` does the exact same thing